pkgs/applications/audio/lsp-plugins/default.nix

```diff
@@ -11,6 +11,14 @@ stdenv.mkDerivation rec {
     url = "https://github.com/sadko4u/${pname}/releases/download/${version}/${pname}-src-${version}.tar.gz";
```

Comment:
This is a different URL to where the patch comes from.  Is that expected?  Should we switch one or the other?

Reply:
No, that confused me at first too, but it's just that the project was split up into these smaller modules starting from 1.2.x (which we already have), and these are now bundled together in the release tarball.